packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -248,7 +257,7 @@ export class IamPolicyChecksWebview extends VueWebview {
                 }
             }
             case 'CloudFormation': {
-                if (document.endsWith('.yaml') || document.endsWith('.yml')) {
+                if (document.endsWith('.yaml') || document.endsWith('.yml') || document.endsWith('.json')) {
```

Comment:
nit: could be a util method.

Reply:
Do we plan to do this after the release? @kevluu-aws

Reply:
I added the necessary util methods